docs/rest-api/dispatch/start-chromium-session.mdx

```diff
@@ -127,6 +127,22 @@ The max number of pixels (width height) is capped at 1920 1080
   Enables dark mode
 </Param>
 
+<Param body="tag" type="string">
+  The `tag` property enforces uniqueness. If a session with tag `"A"` is already running and you attempt to make another
+  session with tag `"A"`, the endpoint will not create a new instance and will instead return the `session_id` and `embed_url` of the existing session.
+</Param>
+
+<Param body="quality" type="object">
+  Used to toggle between sharp mode and smooth mode. Sharp mode increases the streaming bitrate, but requires more bandwidth.
+  <Expandable>
+    <Param body="mode" type="string" default='"smooth"'>
+      Possible values are `"smooth"` or `"sharp"`.
+
+      <Warning>Enabling sharp mode triples the required bandwidth. You can view our bandwidth recommendations in the [Streaming FAQ](/home/faq#streaming).</Warning>
```

Comment:
Can you please mention when you'd want to use sharp mode vs smooth mode?
As a refresher, sharp mode is ideal when reading text is important.
For anything involving movement (videos, dynamic activities), smooth mode is ideal.